### ISSUE TITLE
Reporting period modal: Replace text "Reporting period" with "Program period" in a number of places

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#848]
We received PaQ team feedback that the phrase "Reporting period" is unclear and easily misinterpreted. The request is to use the phrase "program period" instead.

![image](https://user-images.githubusercontent.com/33670923/49174312-d68b4f80-f2fa-11e8-974c-9c6c44e15893.png)

# Acceptance criteria

## Replace text in to-be-renamed "Reporting period modal"
- [x] 1. Modal title is changed to: Program period
- [x] 2. Update intro paragraph:The program period is used in the setup of periodic targets and in Indicator Performance Tracking Tables (IPTT). TolaActivity initially sets the program period to include the program’s official start and end dates, as recorded in the Grant and Award Information Tracker (GAIT) system. The program period may be adjusted to align with the program’s indicator plan.
- [x] 3. At the top of the gray box, change "Reporting start and end dates" to "Program start and end dates"
- [x] 4. Update bottom paragraph: While a program may begin and end any day of the month, program periods must begin on the first day of the month and end on the last day of the month. Please note that the program start date can only be adjusted before targets are set up and a program begins submitting performance results. Because this program already has periodic targets set up, only the reporting end date can be moved later.
     - [x] 4.1. Keep the red highlighting around "before targets are set up and a program begins submitting performance results."
     - [x] 4.2. Make sure any variations on this paragraph are also updated. (I believe the last sentence only appears after targets are set up, and am not sure how to check all variations.)

**Comment by axoplasm** [comment]
> 1. Modal title is changed to: Reporting period

Should this be "Program period"?

**Comment by axoplasm** [comment]
On the Indicators list page, we also have this usage:

![image](https://user-images.githubusercontent.com/169060/49898561-352dee80-fe0e-11e8-8d8f-1e70a3c43b23.png)

**Comment by axoplasm** [comment]
Here's a philosophical Q: is this the *actual* Program start/end date? If so, there’s a case to make that we should refactor the *code* to reflect this reality. 

Or is this a convenience for the sake of users? — i.e. there are still cases where the program start/end dates do not coincide with the reporting dates

**Comment by sanjuroj** [comment]
The latter is correct.  The dates in GAIT should be the official program start and end dates but there are times when users might want to measure over a different time period.  I do wonder if that distinction is lost on users though.  Are we introducing ambiguity by facilitating the association of "Program period" with more than one set of dates?

**Comment by jennyhavah** [comment]
@sanjuroj I've explored this several times with PaQ team advisors and there really doesn't seem to be an obvious, ideal term for what we're trying to convey here. We originally came up with "Reporting period" to differentiate the start/end dates in Tola from the GAIT start end dates driven by the grant cycle. Then we got feedback that "reporting period" might be confused with things like when donor reports are due, or target periods, or what have you. Christie requested "program period" as the least-bad of several less-than-ideal options.
